client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -457,6 +484,11 @@ export const SeoForm = React.createClass( {
 			</Button>
 		);
 
+		const conflictedSeoPlugin = siteIsJetpack
+			// Let's just pick the first one to keep the notice short.
+			? head( this.getConflictingSeoPlugins( activePlugins, siteIsJetpack ) )
```

Comment:
note that although I love the use of `head()` it is significantly faster to use `blah[ 0 ]`